scripts/dcp1-version/populate_dcp_version.py

```diff
@@ -109,11 +109,16 @@ def convert_dcp_version_to_date(version: str):
 if __name__ == '__main__':
     lines = load_lines_from_file(DCP1_GS_FILES_LIST)
     for line in lines:
-        concrete_entity_type, uuid_str, version = find_type_uuid_version(line)
-        collection = COLLECTION_MAP[concrete_entity_type]
-        doc = find_doc_by_uuid(collection, uuid_str)
-        date_obj = convert_dcp_version_to_date(version)
-        update = determine_updates(doc, date_obj)
-        if update:
-            update_dcp_versions(collection, doc, date_obj)
-            LOGGER.info('updated!')
+        LOGGER.info(f'Processing {line}')
+        try:
+            concrete_entity_type, uuid_str, version = find_type_uuid_version(line)
```

Comment:
The "try" bit could go into a function of its own. This way you can test it, and test separately the error handling.

Reply:
done